### PR TITLE
Updated RUGSaar for new place, contributors and dates

### PR DESCRIPTION
**Pull request by SweeD**
As the title says, I updated to our new schedule and place. :smile:

### CODE REVIEW

File: config/whitelabel.yml [hunk 1]
```diff
@@ -142,19 +142,22 @@
...
+      Ligatura
+      Europaallee 27
+      66113 Saarbrücken
     :contributors:
     - :name: Markus Schwed
       :email: ma.schwed@googlemail.com
     - :name: Robert Gogolok
       :email: gogolok@gmail.com
+    - :name: CompanyMood
+      :email: https://www.company-mood.de
```
Comment (bitboxer) [inline config/whitelabel.yml]: This is not an email :wink:

Comment (SweeD) [inline config/whitelabel.yml]: I thought that's okay, because it was already in there for Avarteq (one line beneath) :smile:

Comment (SweeD) [inline config/whitelabel.yml]: Ok, got the problem.  It get linked via `mailto:`  

It's been a while, when i added those lines for Avarteq. 
Give me a sec, I'll take both out. :+1:

Comment (SweeD) [inline config/whitelabel.yml]: Okay, done. 

I changed both to emails, now. :wink:

**Comment by phoet** [comment]
this should go live in a couple of minutes

**Pull request merged**